client/src/components/GameBadge.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import '../containers/App.css';
+
+
+export default class GameBadge extends Component {
+
+    render() {
+        const link="http://megaminerAI.com/" + this.props.gameID;
+				let bgColor = ''
+				// The Background Color of the div should be different based on what the result of the game
+				switch(this.props.result) {
+					case 'Win':
+						bgColor= '#bef5cb'
+						break
+					case 'Lose':
+						bgColor= '#fdaeb7'
+						break
+					default:
+						bgColor= '#cccccc'
+						break
+				}
+
+        return(
+          <div>
+            <div style={{ backgroundColor:bgColor }} className="gamebadge-wrapper col-lg-12">
+
+              <div className="gamebadge-info left">
+                <h3>{this.props.opponentName}</h3>
+              </div>
+
+              <div className="gamebadge-info middle">
+                <h3>{this.props.result}</h3>
+                <p>{this.props.description}</p>
+              </div>
+
+              <div className="gamebadge-info right">
+                <a href={link}>
+<!--  <img src="../images/visit_link.png" alt="visit_link.png" height="85vh" width="85vw"/>  -->
+                <svg width="400" height="180">
+                    <rect x="50" y="20" rx="20" ry="20" width="150" height="150"
+                    style="fill:red;stroke:black;stroke-width:5;opacity:0.5" />
```

Comment:
Style doesn't quite work like this in React. The `style` property takes an object, and you have your keys as the CSS Values.
For example, this style might look like:
```
style={{fill: 'red', stroke: 'black', ... }}
```
(note that the dots aren't part of the code, I just didn't want to write it all out)

This is something you'll run into a lot, so I want to make sure you're understanding how that works. React has a doc page on this, check it out and let me know if you have questions: https://reactjs.org/docs/dom-elements.html#style